packages/tiler/src/tiler.ts

```diff
@@ -55,20 +51,25 @@ export class Tiler {
      * @param zoom WebMercator zoom
      */
     public getRasterTiffIntersection(tiff: CogTiff, x: number, y: number, zoom: number): RasterPixelBounds | null {
-        const extentMeters = tiff.images[0].bbox;
         /** Raster pixels of the output tile */
-        const screenBoundsPx = this.projection.getPixelsFromTile(x, y);
+        const screenPx = this.tms.tileToPixels(x, y);
+        const screenBoundsPx = new Bounds(screenPx.x, screenPx.y, this.tms.tileSize, this.tms.tileSize);
+
         /** Raster pixels of the input geotiff */
-        const tiffBoundsPx = this.projection.getPixelsBoundsFromMeters(extentMeters, zoom);
+        const bbox = tiff.images[0].bbox;
+        const ul = this.tms.sourceToPixels(bbox[0], bbox[3], zoom);
+        const lr = this.tms.sourceToPixels(bbox[2], bbox[1], zoom);
+        const tiffBoundsPx = Bounds.fromUpperLeftLowerRight(ul, lr);
 
```

Comment:
This is something I can use in COG create. Can we make this its own method?

Reply:
It was a method, I am not sure the bbox's are standard though, so dont really want to add it to the `TileMatrixSet` class..

I am then unsure where else to put it.